schwab_api/authentication.py

```diff
@@ -88,62 +48,68 @@ def login(self, username, password, totp_secret=None):
 
         :rtype: boolean
         :returns: True if login was successful and no further action is needed or False
-            if login requires additional steps (i.e. SMS)
+            if login requires additional steps (i.e. SMS - no longer supported)
         """
-        
-        # Log in to schwab using Playwright
-        with self.page.expect_navigation():
-            self.page.goto("https://www.schwab.com/")
+        result = asyncio.run(self._async_login(username, password, totp_secret))
+        return result
+    
+    async def _async_login(self, username, password, totp_secret=None):
+        """ This function runs in async mode to perform login.
+        Use with login function. See login function for details.
+        """
+        self.playwright = await async_playwright().start()
+        if self.browserType == "firefox":
+            self.browser = await self.playwright.firefox.launch(
+                headless=self.headless
+            )
+        else:
+            raise ValueError("Only supported browserType is 'firefox'")
 
+        user_agent = USER_AGENT + self.browser.version
```

Comment:
This variable seems to be unused

Reply:
I'm surprised it worked without it but it's better to use it for robustness. I added it in.